flutter_maps_firestore/lib/main.dart

```diff
@@ -49,7 +49,7 @@ class HomePage extends StatefulWidget {
 }
 
 class _HomePageState extends State<HomePage> {
-  Stream<QuerySnapshot> _iceCreamStores;
+  Stream<QuerySnapshot>? _iceCreamStores;
```

Comment:
Rather than nullable, just mark this late, since it's initialized in `initState`.

Reply:
Sure  @RedBrogdon.